src/schemas/update-proposal.json

```diff
@@ -40,6 +40,14 @@
             "turbo": 1000
           }
         },
+        "labels": {
```

Comment:
No unique items and pattern check on update ?

Reply:
opps missed it, added it now